src/main/webapp/js/reducers/index.jsx

```diff
@@ -1,10 +1,12 @@
 import { combineReducers } from 'redux';
 import vaccineReducer from './vaccinee-reducer';
 import authReducer from './auth-reducer';
+import fieldConfigReducer from './field-config-reducer';
 
 const rootReducer = combineReducers({
   vaccinee: vaccineReducer,
   auth: authReducer,
+  vaccineeFieldConfig: fieldConfigReducer,
```

Comment:
I think it should have different structure eg.
`fieldConfig: {
  vaccinee: [],
  visit: [],
}`
so change the vaccineeFieldConfig to fieldConfig